docs/pages/getting-started.mdx

```diff
@@ -1,329 +1,20 @@
-
-### View Status
-
-```bash
-tsh status
-```
-
-### SSH into a node
-
-```bash
-# list all SSH servers connected to Teleport
-tsh ls
-
-# ssh into `node-name` as `root`
-tsh ssh root@node-name
-```
-
-### Add a node to the cluster
-
-Generate a short-lived dynamic join token using `tctl`:
-
-```bash
-tctl tokens add --type=node
-```
-
-Bootstrap a new node:
-
-<Tabs>
-  <TabItem label="teleport start">
-    Replace `auth_servers` with the hostname and port of your Teleport cluster,
-    `token` with the token you generated above.
-
-    ```bash
-    sudo teleport start \
-    --roles=node \
-    --auth-server=https://teleport.example.com:443 \
-    --token=${TOKEN?} \
-    --labels=env=demo
-    ```
-  </TabItem>
-
-  <TabItem label="cloud-config">
-    Replace `auth_servers` with the hostname and port of your Teleport cluster,
-    `auth_token` with the token you generated above.
-
-    ```ini
-    #cloud-config
-
-    package_upgrade: true
-
-    write_files:
-    - path: /etc/teleport.yaml
-        content: |
-            teleport:
-                auth_token: ""
-                auth_servers:
-                    - "https://teleport.example.com:443"
-            auth_service:
-                enabled: false
-            proxy_service:
-                enabled: false
-            ssh_service:
-                enabled: true
-                labels:
-                    env: demo
-
-    runcmd:
-    - 'mkdir -p /tmp/teleport'
-    - 'cd /tmp/teleport && curl -O https://get.gravitational.com/teleport_(=teleport.version=)_amd64.deb'
-    - 'dpkg -i /tmp/teleport/teleport_(=teleport.version=)_amd64.deb'
-    - 'systemctl enable teleport.service'
-    - 'systemctl start teleport.service'
-    ```
-  </TabItem>
-</Tabs>
-
-### Add an application to your Teleport cluster
-
-Generate a short-lived dynamic token to join apps:
-
-```bash
-tctl tokens add --type=app
-```
-
-Add a new application:
-
-<Tabs>
-  <TabItem label="teleport start">
-    Install Teleport on the target node, then start it using a command as shown below.
-    Review and update `auth-server`, `token`, `app-name`, and `app-uri` before running this command.
-
-    ```bash
-    sudo teleport start \
-    --roles=app \
-    --token=${TOKEN?} \
-    --auth-server=teleport.example.com:3080 \
-    --app-name=example-app  \ # Change "example-app" to the name of your application.
-    --app-uri=http://localhost:8080  # Change "http://localhost:8080" to the address of your application.
-    ```
-  </TabItem>
-</Tabs>
-
-## Guides
-
-Check out our collection of step-by-step guides for common Teleport tasks.
-
-- [Install Teleport](installation.mdx)
-- [Admin Guide](admin-guide.mdx)
-- [Share Sessions](user-manual.mdx#sharing-sessions)
-- [Manage Users](admin-guide.mdx#adding-and-deleting-users)
-- [Github SSO](admin-guide.mdx#github-oauth-20)
-- [Label Nodes](admin-guide.mdx#labeling-nodes-and-applications)
-- [Teleport with OpenSSH](admin-guide.mdx#using-teleport-with-openssh)
+<TileSet>
```

Comment:
Vast improvement! We shouldn't be funneling people into exactly one kind of way to Get Started.

We should also use this approach to create an alternative example for Server Access - two Teleport Nodes on docker.